vuepress/docs/v7.0/tutorials/compose/content-types-tutorial.md

```diff
@@ -4,71 +4,67 @@ sidebarDepth: 2
 
 # Content Types
 
-A Content Type represents or models a Content. Alternatively, a Content is an instance or specialization of a Content Type. 
-
-Content Types are characterized by [Attribute Types](./content-attributes.md). Each Attribute Type consists of one or more properties, known as Attributes, which are defined by their Attribute Type.
+Content is a specialized instance of a content type. A content type provides a template to specify and represent content. 
```

Comment:
specialized -> specific.  A content is an actual instance of a type, like my Honda Odyssey is an instance of a type Car. Rather than specialized instance of Honda minivan. Okay, the analogy isn't perfect.

Reply:
i'm with you 100%. your odyssey is a certain incarnation of a honda, not the hypothetical generic honda that has been souped up for the side shows...